docs/sources/articles/authentication.md

```diff
@@ -1,31 +1,112 @@
-page_title: Running Docker with HTTPS
-page_description: How to setup and run Docker with HTTPS
-page_keywords: docker, docs, article, example, https, daemon, tls, ca, certificate
+page_title: Authenticating to the Docker daemon
+page_description: How to setup the Docker daemon with encryption and authentication
+page_keywords: docker, docs, article, example, https, daemon, tls, ca, certificate, authentication
 
 # Running Docker with https
 
 By default, Docker runs via a non-networked Unix socket. It can also
-optionally communicate using a HTTP socket.
+optionally communicate using a HTTP socket.  Enabling communication
+through HTTP will by default use TLS and require daemon configuration
+to allow client connections.
 
-If you need Docker to be reachable via the network in a safe manner, you can
-enable TLS by specifying the `tlsverify` flag and pointing Docker's
-`tlscacert` flag to a trusted CA certificate.
+
+There are two different ways of authenticating connections between Docker
+client and daemon, both of which use secure TLS connections.
+
+ - **Identity-based authentication** uses an authorized keys list on the daemon
+to whitelist client connections.  The client must also accept the daemon's key
+and remember it for future connections.
+ - **Certificate-based authentication** uses a Certificate Authority to
+authorize connections.  Using this method requires additional setup to enable
+client authentication.
+
+The authentication method is selected using the `--auth` flag with values
+ `identity`, `cert`, or `none` . The `none` method is currently the default but
+`identity` will become the default in a future version.
```

Comment:
I don't think we should speculate about future dev in the docs. We just need to state what things are now.